cmd/skopeo/copy.go

```diff
@@ -192,5 +201,10 @@ var copyCmd = cli.Command{
 			Value: "",
 			Usage: "use docker daemon host at `HOST` (docker-daemon destinations only)",
 		},
+		cli.DurationFlag{
+			Name:  "command-timeout",
+			Value: 30 * time.Minute,
```

Comment:
I don't think this is ever used?

Reply:
the command is used by the jenkins to do image validation and other scan work. The command hangs and cause all the jenkins jobs stuck. 

For general command line usage, it is not very often used.